proto/translate/v1/translate.proto

```diff
@@ -150,6 +151,8 @@ service TranslateService {
     };
   }
 
+  //nolint:lll
```

Comment:
`.proto` is not golang source code, isn't it? `//nolint:` is applicable only the `.go` files.